cli/pkg/kctrl/cmd/app/release/release.go

```diff
@@ -44,6 +44,7 @@ func NewReleaseCmd(o *ReleaseOptions) *cobra.Command {
 	cmd.Flags().StringVar(&o.chdir, "chdir", "", "Working directory with package-build and other config")
 	cmd.Flags().StringVar(&o.outputLocation, "copy-to", defaultArtifactDir, "Output location for artifacts")
 	cmd.Flags().BoolVar(&o.debug, "debug", false, "Print verbose debug output")
+	cmd.Flags().StringVarP(&o.tag, "tag", "t", "", "Tag of the image/bundle to be pushed")
```

Comment:
Let's be specific in the help message and say "Tag pushed with imgpkg bundle (arbitrary tag generated if not specified)"
Since we might be building images are a part of the process it might be misleading to say "image" here

Reply:
Changed it to cmd.Flags().StringVarP(&o.bundleTag, "tag", "t", "", "Tag pushed with imgpkg bundle (default bundle-<TIMESTAMP>)")